pagefaultgames/pokerogue
# [Fix][Sprite] Politoed back sprite

## What are the changes?
 Fixed floating head swirl. Now firmly attached to all versions of Politoed.

## Why am I doing these changes?
Issue reported multiple times by users.

## What did change?
Files changed: Back spritesheet PNG and JSON for default, female, shiny, and shiny female. 

JSON are identical
Position of sprites in sheet is identical
Smaller image size on account of less unique sprites.

### Screenshots/Videos
https://github.com/user-attachments/assets/4fe1b208-ba2b-4fe7-857e-5d19a2753f15


## How to test the changes?
Use `src/overrides.ts` to 
- Set player pokemon to 186
- Set gender female
- Shiny true

There are 4 variations, default, shiny, female, and female shiny. so set the values accordingly to see the change.

NOTE: Couldn't override user pokemon gender for testing. Notice later in the video, the opponent is female but user is not, even though I set both as female. 

## Checklist
